drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java

```diff
@@ -335,6 +345,19 @@ void initialiseContent(final ObservablePath path,
         initialiseAuditLog();
     }
 
+    void initialiseAccess(final boolean isReadOnly) {
+        getAccess().setReadOnly(isReadOnly);
+        getAccess().setHasEditableColumns(canEditColumns());
+    }
+
+    Boolean canEditColumns() {
```

Comment:
```Boolean``` for a reason (vs ```boolean```)? Just asking.. Use of ```Boolean``` in my mind always makes me think this is a tri-value ```boolean``` (```true```, ```false```, ```null```)..

Reply:
Personally, I think that avoiding primitives is more elegant, handling boolean values like any other object in the code base. Additionally (again personally) I try to use enums when something has more than two states. However, I don’t have a strong opinion about this, so I preferred to change it to primitive :-)